tests/sqlparser_common.rs

```diff
@@ -16174,3 +16199,13 @@ fn test_identifier_unicode_start() {
     ]);
     let _ = dialects.verified_stmt(sql);
 }
+
+#[test]
+fn test_odbc_time_date_timestamp_support() {
+    let sql_d = "SELECT {d '2025-07-17'}, category_name FROM categories";
```

Comment:
Could we include a negative test for an expression that doesn't use the expected character? e.g. `SELECT {tt '14:12:01'} FROM foo`